src/main/webapp/i18n/de/lecture.json

```diff
@@ -100,7 +100,14 @@
             "pdfPreview": {
                 "title": "Anhang",
                 "attachmentIDError": "Ungültiger Anhang oder ungültige Anhangs-ID.",
-                "attachmentUnitIDError": "Ungültige Dateieinheit oder ungültige Dateieinheits-ID."
+                "attachmentUnitIDError": "Ungültige Dateieinheit oder ungültige Dateieinheits-ID.",
+                "removePageButton": "Entferne Seiten",
+                "appendFileButton": "Anhänge Datei",
```

Comment:
I would rephrase this to `Datei anhängen`
```suggestion
                "appendFileButton": "Datei anhängen",
```

Reply:
Thanks, changed it :)